plugins/filtering/src/index.ts

```diff
@@ -19,7 +19,7 @@ export default class extends Plugin {
         name: 'LinearFilteringDisplay',
         configSchema,
         stateModel: linearFilteringDisplayModelFactory(configSchema),
-        trackType: 'FeatureTrack',
+        trackType: 'FilteringTrack',
```

Comment:
in this case, this is kind of a placeholder. there is no filteringtrack type but you can set BasicTrack with this renderer

Reply:
the displays "register themselves" with a track type so that they can be listed in as an option in the track menu as an option to change

The lollipop and filtering displays registered themselves with FeatureTrack but in my opinion are not really production grade for general usage. Currently I "removed it" by making it register to non-existent track types because it throws if no trackType is specified IIRC (DisplayType says `(!this.trackType)  throw new Error('no trackType provided for display...')`. 

We could alternatively make it not throw if no track type is given but currently they have to be registered with something so I made them registered to non-existent track types, but they can still be used with FeatureTrack if they are configured to.